cmd/minikube/cmd/config/config.go

```diff
@@ -143,12 +148,12 @@ func ReadConfig() (MinikubeConfig, error) {
 func WriteConfig(m MinikubeConfig) error {
 	f, err := os.Create(constants.ConfigFile)
 	if err != nil {
-		return fmt.Errorf("Could not open file %s: %s", constants.ConfigFile, err)
+		return fmt.Errorf("Cannot open file %s: %s", constants.ConfigFile, err)
 	}
 	defer f.Close()
 	err = encode(f, m)
 	if err != nil {
-		return fmt.Errorf("Error encoding config %s: %s", constants.ConfigFile, err)
+		return fmt.Errorf("Cannot write config %s: %s", constants.ConfigFile, err)
```

Comment:
So why did you switch from "encoding" to "cannot write". This seems to be loosing some information. In the context of error message I think we should keep as much context as possible.